File: C_Sharp_code/Example/WindowsFormsApp/CS_demo15/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CS_demo15
{
    class Student
    {
        private int _no;
        private string _name;
        private string _major;

        public int No
        {
            get
            {
                return _no;
            }
            set
            {
                this._no = value;
            }
        }
        public string Name
        {
            get
            {
                return _name;
            }
            set
            {
                this._name  = value;
            }
        }
        public string Major
        {
            get
            {
                return _major;
            }
            set
            {
                this._major = value;
            }
        }
    }
}

File: C_Sharp_code/Example/WindowsFormsApp/VB_demo16/Student.vb
Public Class Student
    Private _no As Integer
    Private _name As String
    Private _major As String

    Public Property No() As Integer
        Get
            Return _no
        End Get
        Set(value As Integer)
            Me._no = value
        End Set
    End Property
    Public Property Name() As String
        Get
            Return _name
        End Get
        Set(value As String)
            Me._name = value
        End Set
    End Property
    Public Property Major() As String
        Get
            Return _major
        End Get
        Set(value As String)
            Me._major = value
        End Set
    End Property

End Class
